doc/sphinx-guides/source/api/native-api.rst

```diff
@@ -3005,6 +3058,18 @@ List all role assignments of a role assignee (i.e. a user or a group)::
 
 Note that ``identifier`` can contain slashes (e.g. ``&ip/localhost-users``).
 
+.. _remove-role-assignments:
```

Comment:
I'm not sure we really need this API anymore. But there's also no harm in keeping it.

Reply:
OK, for discussion. I'm not sure this is an API right now? Maybe just available through the Dashboard? We can discuss use cases for disable vs. remove all roles and determine what to keep.